contracts/exchange-libs/test/lib_order.ts

```diff
@@ -0,0 +1,75 @@
+import { addressUtils, blockchainTests, constants, describe, expect } from '@0x/contracts-test-utils';
+import { assetDataUtils, orderHashUtils } from '@0x/order-utils';
+import { Order } from '@0x/types';
+import { BigNumber, providerUtils, signTypedDataUtils } from '@0x/utils';
+import * as ethUtil from 'ethereumjs-util';
+
+import { artifacts, TestLibOrderContract } from '../src';
+
+blockchainTests('LibOrder', env => {
+    let libOrderContract: TestLibOrderContract;
+    let order: Order;
+    let chainId: number;
+    before(async () => {
+        libOrderContract = await TestLibOrderContract.deployFrom0xArtifactAsync(
+            artifacts.TestLibOrder,
+            env.provider,
+            env.txDefaults,
+        );
+        chainId = await providerUtils.getChainIdAsync(env.provider);
```

Comment:
```suggestion
        chainId = await env.getChainIdAsync();
```
But, also, why even get the env's chainId? Doesn't seem to matter what this value is so long as it isn't `1337`.